src/main/webapp/WEB-INF/category.jsp

```diff
@@ -27,6 +27,13 @@
       #map {
         height: 75px;
       }
+      <% for(int i = 0; i < messages.size(); i++) { %>
+      		<%if(!messages.get(i).getLat().isEmpty()){%>
+      		#map<%=i%>{
```

Comment:
You shouldn't need to add this for each id. You should just define a class instead and use that.

Just use something like the following here w/o the loop.
.message_map {
  height: 75px; 
}

and then put the following in all your map divs
  class="message_map"

Reply:
Done